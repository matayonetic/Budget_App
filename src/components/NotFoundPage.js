import React from "react";
import { Link } from "react-router-dom";

// 404 Page
const NotFoundPage = () => {
  return (
    <div>
      <h2>Oops! It's a 404</h2>
      <p>
        <Link to={"/"}>Go Home</Link>
      </p>
    </div>
  );
};

export { NotFoundPage as default };
