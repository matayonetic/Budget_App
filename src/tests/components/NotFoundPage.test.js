import React from "react";
import { shallow } from "enzyme";
import NotFoundPage from "../../components/NotFoundPage";

// Render Page
test("Should render NotFoundPage correctly", () => {
  const wrapper = shallow(<NotFoundPage />);
  expect(wrapper.find("h2").text()).toBe("Oops! It's a 404")
  expect(wrapper).toMatchSnapshot();
});
