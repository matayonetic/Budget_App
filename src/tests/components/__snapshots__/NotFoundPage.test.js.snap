// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render NotFoundPage correctly 1`] = `
<div>
  <h2>
    Oops! It's a 404
  </h2>
  <p>
    <Link
      to="/"
    >
      Go Home
    </Link>
  </p>
</div>
`;
